qiita_pet/handlers/rest/study_samples.py

```diff
@@ -27,6 +28,58 @@ def get(self, study_id):
         self.write(json_encode(samples))
         self.finish()
 
+    @authenticate_oauth
+    def patch(self, study_id):
+        study = self.safe_get_study(study_id)
+        if study is None:
+            return
+
+        if study.sample_template is None:
+            self.fail('No sample information found', 404)
+            return
+        else:
+            sample_info = study.sample_template.to_dataframe()
+
+        data = pd.DataFrame.from_dict(json_decode(self.request.body),
+                                      orient='index')
+
+        if len(data.index) == 0:
+            self.fail('No samples provided', 400)
+            return
+
+        categories = set(study.sample_template.categories())
+
+        if set(data.columns) != categories:
```

Comment:
Yes, I think this is correct @antgonza, as per @josenavas's request, we only need to add blank samples right now, and AFAIK they will inherit the existing metadata categories in the sample information.